source/includes/reference/_app_webhooks.md

```diff
@@ -0,0 +1,209 @@
+# Application Webhooks
+
+Application webhook subscription is a mechanism that will allow you to receive notifications to your servers whenever various events happen in relation to different resources created by an ***application***. You can subscribe to receive webhooks about resource created under a specific [***profile***](#profile-webhooks).
+
+Before proceeding, make sure the endpoint where you intend to receive webhooks satisfies the following requirements:
+
+* Has a valid domain name (IPs are disallowed)
+* Listens to HTTPS requests on port `443`
+* Has a valid HTTPS certificate signed by a trusted Certificate Authority - CA (self-signed or expired certificates are not accepted)
+* Does not include any query arguments in the URL
```

Comment:
Do we actually check that?

Reply:
Yes, https://github.com/transferwise/webhook-sender/blob/master/src/main/java/com/transferwise/webhook/services/callback/StrictUrlValidationService.java#L29